change/@ni-nimble-components-4381b5e1-bbcc-48b4-9751-38f3ac8f2aa8.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
I'd also be okay marking all these as none since they are test changes. The tsconfig changes shouldn't have generated js impacts (but I didn't check)
Technically nimble components has a README change which is package contents so could be patch and dependent change type none.
Our just leave em as patch 🤷